help/saml-authentication.md

```diff
@@ -213,6 +213,53 @@ providers.
 
     Once SAML has been configured, consider also [configuring SCIM](/help/scim).
 
+## Synchronizing group membership with SAML
+
+You can configure each Zulip user's [groups](/help/user-groups) be updated based
+on their groups in your identity provider's (IdP's) directory every time they
+log in.
```

Comment:
Maybe add "Because SAML only delivers claims to the Zulip server when a user actually logs in, group membership changes in the IdP will not be reflected in Zulip until that user next logs in via SAML. SCIM group sync is the recommended alternative to SAML group sync for installations that wish to avoid this limitation.".

We can link to the open issue/PR for the "SCIM group sync" terms for now, and update the text to link to the docs when that happens?

Reply:
OK. I'm imagining SCIM group sync will merge in like a week, but I guess we can do it that way.

@mateuszmandera BTW I'm assuming you are managing next steps on #35162 -- either pulling it into the main PR (since I made some edits) or fixing it up, probably including some temporary nocoverage comments for the code that's not yet called anywhere so it can merge.